src/WormholeJoin.sol

```diff
@@ -75,6 +75,16 @@ contract WormholeJoin {
         uint248 pending;
     }
 
+    // --- Errors ---
+    error NotAuthorized(address sender, uint256 wards);
+    error FileUnrecognizedParam(bytes32 what);
+    error NotAllowedBiggerInt256(uint256 data);
```

Comment:
I think we can generalize this to `Overflow(uint256 num)`.

Reply:
done in 0172295